src/open_dread_rando/door_locks/custom_door_types.py

```diff
@@ -23,15 +22,14 @@
 
 
 @functools.cache
-def _template_read_shield(file: str):
-    with templates_path().joinpath(f"{file}.json").open() as f:
-        return json.load(f)
+def _template_read_shield(file: str) -> bytes:
+    return templates_path().joinpath(file).read_bytes()
 
 
 class DoorTemplates(Enum):
-    HEXAGONS = "template_doorshield_hexs_bmsad"
```

Comment:
change these to `actors/props/doorshieldmissile/charclasses/doorshieldmissile.bmsad`, ...